src/dswx_sar/mosaic_gcov_frame.py

```diff
@@ -756,13 +867,16 @@ def run(cfg):
     # Mosaic input RTC into output Geotiff
     reader.process_rtc_hdf5(
         input_list,
-        scratch_dir,
+        output_dir,
```

Comment:
I think this line is not what we want to do. The mosaic geotiff should be the scratch directory. The output dir will be used only for saving the final (water) products. Please refer to the PR I submitted to your repo.

Reply:
@oberonia78 I think I will remove all 'output_dir'.